### ISSUE TITLE
Broken on OSX El Capitan (10.11.2)

### ISSUE THREAD

**Issue by trasalghul** [netPark/node-openalpr#4]
Installed openalpr from brew, getting

``` javascript
  dyld: lazy symbol binding failed: Symbol not found: __ZN4alpr4AlprC1ESsSsSs
  Referenced from: /Users/cipher/WebstormProjects/reporty/node_modules/node-openalpr/release/darwin_x64/node_openalpr.node
  Expected in: dynamic lookup

dyld: Symbol not found: __ZN4alpr4AlprC1ESsSsSs
  Referenced from: /Users/cipher/WebstormProjects/reporty/node_modules/node-openalpr/release/darwin_x64/node_openalpr.node
  Expected in: dynamic lookup

[1]    59801 trace trap  node app.js
```

gcc -v output

```
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 7.0.2 (clang-700.1.81)
Target: x86_64-apple-darwin15.2.0
Thread model: posix
```

**Comment by kjlaw89** [comment]
This is likely related to updating the header files for OpenALPR 2.2. Try to rebuild everything again and see if it works for you.

**Comment by hyuni** [comment]
@trasalghul this issue fixed PR #9
